catroid/src/org/catrobat/catroid/content/bricks/CameraBrick.java

```diff
@@ -165,7 +165,7 @@ public View getViewWithAlpha(int alphaValue) {
 
 	@Override
 	public int getRequiredResources() {
-		return Brick.NO_RESOURCES;
+		return Brick.VIDEO;
```

Comment:
Why do you need a separate bit mask for video? wouldn't it work with CAMERA_BACK || CAMERA_FRONT?